### PR TITLE
Enable `golint` on the tsdb/engine/wal

### PR DESCRIPTION
**Pull request by gabelev**
Cleaned up tsdb/engine/wal.go and wal_test.go to enable golint per issue #4098.

- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

### CODE REVIEW

**Comment by e-dard** [comment]
@gabelev can you update the `CHANGELOG`?

**Comment by gabelev** [comment]
@e-dard: done.

File: tsdb/engine/wal/wal.go [hunk 1]
```diff
@@ -243,6 +245,7 @@ func (l *Log) Open() error {
 	return nil
 }
 
+// DiskSize will returns the current size of a Log file.
```
Comment (corylanou) [inline tsdb/engine/wal/wal.go]: nit: `returns` -> `return`

**Comment by corylanou** [comment]
One minor nit.  Otherwise +1. Will need a rebase as well.  Thanks!

**Comment by gabelev** [comment]
@corylanou: made the small fix. rebased on master.

**Comment by e-dard** [comment]
@gabelev needs another rebase

**Comment by gabelev** [comment]
@e-dard: ok. rebased again.

**Pull request merged**